app/models/parameter.rb

```diff
@@ -9,7 +9,12 @@ class Parameter < ActiveRecord::Base
   include Authorizable
   validates :name, :presence => true, :no_whitespace => true
 
-  scoped_search :on => :name, :complete_value => true
+  def self.inherited(child)
+    child.instance_eval do
+      scoped_search :on => :name, :complete_value => true
```

Comment:
Thanks, I've removed the search definition from CommonParameter and added the :value definition here.